examples/next/src/components/BannerDemo.tsx

```diff
@@ -0,0 +1,14 @@
+import { VStack } from "@animareflection/ui";
+import { Banner } from "@animareflection/ui/client";
+
+import { Wrapper } from "components";
+
+const BannerDemo = () => (
+  <Wrapper title="Banner">
+    <VStack position="relative" inset={0}>
+      <Banner closable>Banner</Banner>
+    </VStack>
+  </Wrapper>
+);
+
+export default BannerDemo;
```

Comment:
Add `relative` positioning to the `Banner` recipe (in the base), then you can simplify this:

```ts
import { Banner } from "@animareflection/ui/client";

import { Wrapper } from "components";

const BannerDemo = () => (
  <Wrapper title="Banner">
    <Banner closable>Banner</Banner>
  </Wrapper>
);

export default BannerDemo;
```

After that is done it all lgtm so should be able to send it!